docs/howtos/locally-published-components-in-ios.md

```diff
@@ -0,0 +1,18 @@
+# Using locally-published components in Firefox for iOS
+
+It's often important to test work-in-progress changes to this repo against a real-world
+consumer project. Here are our current best-practices for approaching this on iOS:
+
+1. Make a local build of the application-services framework using `./build-carthage.sh`.
+1. Checkout and `carthage bootstrap` the consuming app (for example using [these instructions with Firefox for
```

Comment:
Cool `1.` trick 👍

Reply:
I learned about this very recently and have been busily applying it everywhere!

Reply:
oooh. TIL